src/hypothesis/internal/debug.py

```diff
@@ -42,3 +45,28 @@ def minimal(
         settings=settings,
         random=random,
     )
+
+
+HYPOTHESIS_ROOT = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), '..'))
```

Comment:
I usually put the `abspath()` straight around the `__file__`, as that can return a relative or an absolute path. Then this becomes:

```python
HYPOTHESIS_ROOT = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

I’m not sure it makes a difference in this case, but I’m also not sure it doesn’t.